lib/braveAlerter.js

```diff
@@ -98,32 +98,32 @@ class BraveAlerter {
     }
   }
 
-  async sendFallbackMessageForSession(alertInfo) {
+  async sendFallbackMessagesForSession(alertInfo) {
     try {
       const alertSession = await this.getAlertSession(alertInfo.sessionId)
 
       if (alertSession.alertState === ALERT_STATE.WAITING_FOR_REPLY) {
-        // Send fallback alert
-        if (alertInfo.fallbackToPhoneNumber && alertInfo.fallbackFromPhoneNumber) {
-          const response = await twilio.sendTwilioMessage(
-            alertInfo.fallbackToPhoneNumber,
-            alertInfo.fallbackFromPhoneNumber,
-            alertInfo.fallbackMessage,
-          )
-
-          if (response) {
+        // Send fallback alert(s)
+        if (alertInfo.fallbackToPhoneNumbers && alertInfo.fallbackFromPhoneNumber) {
+          const promises = []
+          alertInfo.fallbackToPhoneNumbers.forEach(fallbackToPhoneNumber => {
+            promises.push(twilio.sendTwilioMessage(fallbackToPhoneNumber, alertInfo.fallbackFromPhoneNumber, alertInfo.fallbackMessage))
```

Comment:
is this the same idea as we had in the Sensors state machine, instead of waiting for each promise to resolve in sequence you put them out all at once and then await the whole lot of them>?

Reply:
Yes, that's right. I took the idea from Sensors. I know that you recently took it out, but in this case I think it's pretty useful because we have an unknown number of fallback numbers that we want to wait for. What do you think?

Reply:
makes sense!